doc/rbd/rbd-mirroring.rst

```diff
@@ -216,6 +216,17 @@ peer cluster connection attributes when adding a mirroring peer, use the
           UUID                                 NAME   CLIENT                 MON_HOST                KEY                                      
           587b08db-3d33-4f32-8af8-421e77abb081 site-b client.rbd-mirror-peer 192.168.1.1,192.168.1.2 AQAeuZdbMMoBChAAcj++/XUxNOLFaWdtTREEsw== 
 
+.. important:: If Ceph returns the error message ``rbd: failed to import peer
+   bootstrap token``, then you might need to set both pools so that they have
+   the same name. See :ref:`Renaming a Pool<rados_renaming_a_pool>` for
+   instructions on renaming pools.
+
+   In one test in June of 2025, it was impossible to achieve mirroring between
+   a ``site-a`` with Ceph version 17.2.8 installed and a ``site-b`` with Ceph
+   version 17.2.6 installed unless the pools had the same name. This
+   information came from Petr Tlapa, and was reported through the upstream Ceph
+   community.
```

Comment:
I'd omit the paragraph entirely.  The date and who performed the test isn't really relevant because the pool names have always had to be the same -- ever since RBD mirroring was introduced in 2016.

Reply:
Usually I like to credit the members of the upstream community, but because pool names have always had to be the same since the introduction of RBD mirroring, I'm convinced to remove this paragraph.